src/schema/mutation/duplicateApplication.mutation.ts

```diff
@@ -27,10 +28,14 @@ export default {
     graphQLAuthCheck(context);
     try {
       const user = context.user;
-
       const ability: AppAbility = context.user.ability;
-      if (ability.can('create', 'Application')) {
-        const baseApplication = await Application.findById(args.application);
+      const filters = Application.find(
```

Comment:
@RenzoPrats 
this part is good, but you still need to check if the user can create an application
i'll add that in a new ticket